xapian-core/include/xapian/weight.h

```diff
@@ -449,20 +454,38 @@ class XAPIAN_VISIBILITY_DEFAULT TfIdfWeight : public Weight {
      *     @li 'p': Prob    idfn=log((N-Termfreq)/Termfreq)
      *     @li 'f': Freq    idfn=1/Termfreq
      *     @li 's': Squared idfn=log(N/Termfreq)^2
+     *     @li 'P': Pivoted idfn=log((N+1)/Termfreq)
      *
      * @li The third and the final character indicates the normalization for
      *     the document weight.  The following normalizations are currently
      *     supported:
      *
      *     @li 'n': None wtn=tfn*idfn
+     *     @li 'P': Pivoted wtn=wqf(tfn*idfn*(1-s+(s*normlen))+delta*idfn)
+     *              Note: Valid parameters s and delta values should be non-negative
+     *                    & non-zero when using pivoted normalization string("PPP")
```

Comment:
Ah, okay. I think we definitely are going to need an advanced document in the getting started guide, similar to the one for tuning `LMWeight`, which explains all these different variants, points out that `"PPP"` is Piv+, and (hopefully) gives some idea of when you might use the different ones.